apps/web/src/components/Basenames/RegistrationForm/RegistrationButton.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames';
+import { ConnectWallet, ConnectWalletText } from '@coinbase/onchainkit/wallet';
+import { Button, ButtonSizes, ButtonVariants } from 'apps/web/src/components/Button/Button';
+import { useAccount } from 'wagmi';
+
+type RegistrationButtonProps = {
+  correctChain: boolean;
+  registerNameCallback: () => void;
+  switchToIntendedNetwork: () => void;
+  insufficientFundsNoAuxFundsAndCorrectChain: boolean;
+  registerNameIsPending: boolean;
+};
+
+export function RegistrationButton({
+  correctChain,
+  registerNameCallback,
+  switchToIntendedNetwork,
+  insufficientFundsNoAuxFundsAndCorrectChain,
+  registerNameIsPending,
+}: RegistrationButtonProps) {
+  const { isConnected } = useAccount();
+
+  if (!isConnected) {
+    return (
+      <ConnectWallet
+        className={classNames(
+          'bg-button-black text-white hover:bg-button-blackHover active:bg-button-blackActive', // ButtonVariants.Black
+          'px-10 py-3.5 text-sm md:text-lg', // ButtonSizes.Medium
```

Comment:
imo these should prob be standardized and exported if they correspond to a theme

Reply:
these are standardized and exported from [Button](https://github.com/base/web/blob/master/apps/web/src/components/Button/Button.tsx). 

Unfortunately, the exported variables do not override OnchainKit's default styles. So I had to copy over the hardcoded values.